packages/cashier/src/pages/payment-agent/payment-agent-list/payment-agent-list.tsx

```diff
@@ -53,6 +76,7 @@ const PaymentAgentList = ({
 
             setSideNotes(side_notes);
         }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@hamid-deriv Can’t just add `setSideNotes` in the dependency array?

Reply:
@farzin-fs It has a performance issue (exceeding render limit) because the `setSideNotes` function gets rerendered in the vertical-tab component and by adding this to the dependency array the page will go into an infinite loop.